src/revChatGPT/V1.py

```diff
@@ -336,9 +330,9 @@ def login(self) -> None:
             error = t.AuthenticationError("Insufficient login details provided!")
             raise error
         auth = Authenticator(
-            email_address=self.config.get("email"),
-            password=self.config.get("password"),
-            proxy=self.config.get("proxy"),
+            email_address=self.config.get("email", ""),
+            password=self.config.get("password", ""),
+            proxy=self.config.get("proxy", ""),
```

Comment:
This is broken. The OpenAIAuth library needs to be translated from Go to Python.

Reference: https://github.com/acheong08/OpenAIAuth

*todo note for myself